cmd/intercept/intercept.go

```diff
@@ -125,6 +127,8 @@ func main() {
 				msgAccession: "accessionIDs",
 				msgIngest:    "ingest",
 				msgMapping:   "mappings",
+				msgRelease:   "mappings",
+				msgDeprecate: "mappings",
```

Comment:
```suggestion
				msgRelease:   "release",
				msgDeprecate: "deprecate",
```
`msgRelease` and `msgDeprecate` should probably be `release` and `deprecate`, but not sure, since there are similar things many times (singular, plural, prefix-name).

If I understand correctly, we should use these constants https://github.com/neicnordic/sda-pipeline/blob/d4eefc99c44652efe1ea61bde9ae95a5012de496/cmd/intercept/intercept.go#L16-L23 instead of building the dictionary here https://github.com/neicnordic/sda-pipeline/blob/d4eefc99c44652efe1ea61bde9ae95a5012de496/cmd/intercept/intercept.go#L126-L132 and here https://github.com/neicnordic/sda-pipeline/blob/d4eefc99c44652efe1ea61bde9ae95a5012de496/cmd/intercept/intercept.go#L161-L168

Reply:
nope this tells to which queue the type of message: `mapping`, `release` and `deprecate` should go to, which is the `mappings` queue

Reply:
no, those consts are used to map message types with json schema corresponding in the dictionary, I am not sure exactly why it was built like that, and that is not the scope of this issue, just following what was done.